src/saturn_engine/worker_manager/context.py

```diff
@@ -24,21 +24,23 @@ def static_definitions(self) -> StaticDefinitions:
         return self._static_definitions
 
     def load_static_definition(self, session: AnySession) -> None:
-        self._static_definitions = _load_static_definition(
-            session=session, config=self.config
-        )
+        static_definition = _load_static_definition(session=session, config=self.config)
+        if static_definition:
+            self._static_definitions = static_definition
 
 
 def _load_static_definition(
     config: WorkerManagerConfig, session: AnySession
-) -> StaticDefinitions:
+) -> StaticDefinitions | None:
     """
     Static definitions contain objects defined in a declarative configuration:
     - Inventories
     - Topics
     - Jobs
     - JobDefinitions
     """
+    if not config.static_definitions_directories:
```

Comment:
I feel it would be nicer if we return a `StaticDefinitions`. Plus it has a default initializer, so you can just do `return StaticDefinitions()`